draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -333,6 +345,27 @@ The classic congestion control algorithm {{!RFC5681}} and the widely deployed
 Cubic algorithm {{?RFC9438}} do not address it, but a new
 congestion control algorithm has the opportunity to improve the state of the art.
 
+### Protection Against High Packet Loss
+
+A congestion control algorithm should reduce its sending
+rate if experiencing high packet loss.
+Even if a congestion control algorithm can tolerate high loss, many types
+of short request/response traffic do not want to pay the loss recovery
+performance penalty induced by high loss rates.
+
+The first version of the BBR algorithm {{BBRv1-draft}} failed this requirement.
+Experimental evaluation {{BBRv1-Evaluation}} showed that
+it caused a sustained rate of
+when multiple BBRv1 flows share a bottleneck and the buffer size was
+less than roughly one and a half BDP.
+This kind of behavior needed to be fixed, and indeed
+further versions of BBR {{BBR-draft}}
+fixed this issue.
+
```

Comment:
@gorryfair you wrote "I don't agree", but from the remainder of your text I suppose this is a typo.

The problem with evaluation is that there is a vast space of possible network conditions. For BBRv1, the issue was manifest with a specific configuration and a specific size of bottleneck buffers. For other protocol, we might find issue  with a range of delays, or with a specific number of competing flows.

We probably need a good sentence for "try enough combination because the worms may hide under an not-yet-turned rock".